src/wrappers/themis/jsthemis/test/test.js

```diff
@@ -307,6 +308,22 @@ describe("jsthemis", function(){
                 expect_code(addon.INVALID_PARAMETER)
             )
         })
+        it('does not allow EC public key', function() {
```

Comment:
I didn't add EC private since they are already involved in every Secure Session test that tries to encrypt and send data. Put :+1: if you think it's still better to add them (to all wrappers mentioned in this PR + jsthemis) or :-1: if I convinced you we don't need them.
Anyway, users are not like "oh, let's read some tests, here's private EC key and it doesn't raise exception so I can use it and here's private RSA and looks like I won't be able to use it", they read the docs instead.

Reply:
1. That's not for users, that's for future us. Future us might forget that EC private key is a valid case (as we forgot about RSA keys in SS).
2. These are different types of tests.

While I agree that adding additional tests for EC private keys overlaps a bit with current test suite, I don't see any disadvantages of this approach, only improvements of maintainability.